pkg/kvstore/etcd.go

```diff
@@ -87,6 +90,16 @@ var (
 	etcdDummyAddress = "http://127.0.0.1:4002"
 
 	etcdInstance = newEtcdModule()
+
+	// Stop using a package scoped logger. We need to pass each etcd client a
```

Comment:
Where is the package scoped logger? Shouldn't we remove it in favor of this one? I don't see any lines removing the existing logger.